src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import cn from 'classnames';
+
+import { FilterField } from '../../types/FilterField';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filterField: FilterField;
+  setFilterField: (field: FilterField) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterField,
+  setFilterField,
+}) => {
+  const activeTodos = todos.reduce(
```

Comment:
Just to be more precise, the variable can be named as 'activeTodosCount' or something.